apps/src/code-studio/pd/facilitator_program_registration/FacilitatorProgramRegistration.jsx

```diff
@@ -0,0 +1,199 @@
+import React from 'react';
```

Comment:
More generally speaking - we should consider making general form classes. This seems reusable